pages/components/grid/grid.css

```diff
@@ -0,0 +1,55 @@
+.grid {
+    display: grid;
+    gap:0.75rem;
+    width: 45rem;
+    height: 7.5rem;
+}
+
```

Comment:
It will be better if you keep you margin and padding away from your base class. Doing this way it will be applied wherever you use `.grid` class.